docs/utilities/recipes.md

```diff
@@ -72,11 +73,13 @@ Item Objects
 
 Type
 ----
-TODO
-    "type": "forge:oreshaped",
+Both Minecraft and Forge require you to specify the type of your recipe within the type tag. You can think of this as the definition of which crafting layout is to be used, for example `minecraft:crafting_shaped` or `minecraft:crafting_shapeless` or `minecraft:smelting` for the furnace. Forge has also specified `forge:oreshaped` in order to use the ore dictionary within a shaped recipe. The above tags are example tags only and not a complete list of all available options for the type!
```

Comment:
forge:oreshaped has no use anymore since there is an oredict ingredient. I'd just leave it out entirely, or even note its uselessness.

Reply:
Probably the latter. Lots of tutorials are having people use the Forge ones and it would be nice to have a place to point at saying they aren't useful.

Reply:
@tterrag1098 @TehNut Should I leave it this way then or to what should I change this (and what's with line 20)? I'm also wondering whether there is any information about the oredict ingredient, as I can't remember having written or read anything about this.